packages/edit-site/src/components/sidebar-navigation-screen-page/page-details.js

```diff
@@ -138,26 +139,21 @@ export default function PageDetails( { id } ) {
 		[ record?.parent ]
 	);
 	return (
-		<VStack spacing={ 5 }>
```

Comment:
Apologies, I missed this earlier, but I think the removal of `spacing` here caused a regression for the Details section in the Pages screen:

| Before | After |
| --- | --- |
| <img width="348" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/cb32c75f-0ed9-41a8-86b7-fd6900ec4a7a"> | <img width="352" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/cb9d0757-bc71-48a4-aff1-93135bb7713e"> |

Could we add back in spacing (or the ability to pass down a value for it to `SidebarNavigationScreenDetailsPanel` in the follow-up PR in https://github.com/WordPress/gutenberg/pull/51669?

Reply:
Good spotting. Done.